chart/Chart.yaml

```diff
@@ -27,7 +27,6 @@ appVersion: "0.0.0"
 dependencies:
   - name: crds
     version: "0.0.0"
-    condition: crds.enabled
```

Comment:
this might still be useful if we want to disable the chart itself?

Reply:
Hmm, okay, it's fine to leave it in I guess.